src/mo_frontend/typing.ml

```diff
@@ -2994,6 +3030,8 @@ and object_of_scope env sort dec_fields scope at =
         | Some src ->
           Field_sources.add_src env.srcs src.id_region;
           T.{lab = id; typ = T.Typ c; src = {depr = src.depr; track_region = src.id_region; region = src.field_region}}::tfs
+        | _ when sort = T.Mixin ->
```

Comment:
Ok, I guess this is just adding all types, not just public ones, in the case of a mixin?